### DISCUSSION TITLE
Resolving a merge conflict using the command line does not seem to work

### DISCUSSION THREAD

**Discussion by rdmccart** [community/community#81041] [New to GitHub]
I created a branch using git bash, added a file, committed, and pushed to origin with no errors or conflicts. When I went to github I had 23 conflicts. How does this happen? Why does git allow me to commit and github tells me there are conflicts? Shouldn't git tell me this before?

I'm not even sure how to resolve the conflicts. I followed the steps here "https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line" and step 3 doesn't work. When I use git status in git bash it says "nothing to commit, working tree clean". How can that be if github is telling me there are conflicts?

**Comment by varunpats** [comment]
Hey, I always take a latest pull from git before committing any changes. That helps to find out and resolve conflicts on code editor.

Take a latest pull from the branch it will show conflicts on code editor, which I think is a better place to resolve conflicts.

**Reply by rdmccart** [reply]
Thanks for the response but I always use git fetch and git pull origin main before creating a new branch so I'm still not sure why this is happening.
